buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -267,7 +267,10 @@ try {
                                     build(job_name, REPO, BRANCH, SHAS, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, SPEC, SDK_VERSION, BUILD_NODE, TEST_NODE, EXTRA_GETSOURCE_OPTIONS, EXTRA_CONFIGURE_OPTIONS, EXTRA_MAKE_OPTIONS, OPENJDK_CLONE_DIR, ADOPTOPENJDK_REPO, ADOPTOPENJDK_BRANCH, AUTOMATIC_GENERATION, CUSTOM_DESCRIPTION)
                                 }
                             }
-                        }
+                        }   
+                    }
+                    if (params.SLACK_ON_SUCCESS && SLACK_CHANNEL) {
+                        slackSend channel: SLACK_CHANNEL, color: 'good', message: "Build Passed: Started by ${JOB_NAME} #${BUILD_NUMBER} (<${BUILD_URL}|Open>)"
```

Comment:
I don't think we need the `Started by`, other than that it looks good to me.

Reply:
Also, this is in the wrong spot. It should be after the promote section. Where it is now, the message is sent right after the setup is complete before anything has run.

Reply:
> Also, this is in the wrong spot. It should be after the promote section. Where it is now, the message is sent right after the setup is complete before anything has run.
Thanks for reviewing it! I will amend it. :)